lib/std/math.zig

```diff
@@ -272,25 +275,44 @@ pub const gcd = @import("math/gcd.zig").gcd;
 pub const gamma = @import("math/gamma.zig").gamma;
 pub const lgamma = @import("math/gamma.zig").lgamma;
 
-/// Sine trigonometric function on a floating point number.
+/// Absolute value function on a real or complex number.
 /// Uses a dedicated hardware instruction when available.
-/// This is the same as calling the builtin @sin
+/// For real numbers, this is the same as calling the builtin @abs
+pub inline fn abs(value: anytype) complex.ScalarType(@TypeOf(value)) {
+    return switch (@TypeOf(value)) {
+        Complex(f32), Complex(f64) => complex.abs(value),
```

Comment:
Did you consider `Complex(f128)` and `Complex(comptime_float)`?

Reply:
You didn't mark `isComplex` as inline.